disposable_email_blacklist.conf

```diff
@@ -1544,6 +1560,7 @@ secretemail.de
 secure-mail.biz
 secure-mail.cc
 secured-link.net
+securehost.com.es
```

Comment:
You're right, the host doesn't seem to be a disposable domain host. Maybe the domain was sold to a new owner and they use it now for hosting their website. I'm going to remove this one.